robottelo/host_helpers/api_factory.py

```diff
@@ -628,12 +628,6 @@ def __init__(self, temp):
         :param entities.ProvisioningTemplate temp: entities.ProvisioningTemplate's object
         """
         self.temp = temp
-        if not isinstance(self.temp, self._satellite.api.ProvisioningTemplate):
```

Comment:
We need to keep this check.

Reply:
could you provide some reason for me please?